openmrs/apps/clinical/app.json

```diff
@@ -29,6 +29,10 @@
         "quickPrints":false,
         "allowAdhocTeleConsultation": false,
         "teleConsultationDomain": "meet.jit.si",
+        "terminologyService": {
+            "system": "SNOMED",
+            "limit": 30
```

Comment:
what does limit indicate here? 
also does the backend interpret the system? typically in FHIR the default system for SNOMED is http://snomed.info/sct

Reply:
The limit parameter indicates the maximum number of responses that can be returned in a single request. 
For the "system" refers to the "SOURCE" field in mapping. Maybe it would be better to rename it to avoid confusion
<img width="618" alt="Screenshot 2023-05-15 at 13 32 24" src="https://github.com/Bahmni/snomed-default-config/assets/48393059/8f7c1f26-57e5-4aef-afed-63309deccabd">

Reply:
Why is source needed? Is it for the FORM Builder to decide which terms to display on the screen? I am guessing this call is going DIRECTLY to snowstorm, and snowstorm sends a response back from which the formbuilder UI needs to select/show results.

Reply:
When an answer is selected and saved, the saved concept UUID and the UUIDs from the snowstorm server are different.  I use the mapping data to pre-select the saved selections.

Reply:
I see. Since on save, there is a reference map being created for the concept, you need to know which reference map you need to read from OpenMRS Concept Dictionary for this term.. and for that you use the "system" attribute to know that you need to read the SNOMED mapping reference for the concept. Is this correct?

Reply:
Yes